src/components/course/data/tests/utils.test.jsx

```diff
@@ -984,13 +1010,14 @@ describe('getMissingApplicableSubsidyReason', () => {
     const mockData = {
       ...baseMockData,
       catalogsWithCourse: ['test-catalog-uuid'],
-      customerAgreementConfig: {
+      customerAgreement: {
         subscriptions: [
```

Comment:
nit: I don't believe the `customerAgreement` has a list of `subscriptions` anymore. It does have a `availableSubscriptionCatalogs` returned now though (list of catalog uuids).